retriever/lib/fetch.py

```diff
@@ -0,0 +1,30 @@
+
+import os
+import sqlite3
+import pandas as pd
+from retriever.engines import sqlite
+from retriever.lib.defaults import HOME_DIR
+from retriever.lib.install import install_sqlite
+from retriever.lib.download import download
+
+
+
+def fetch(dataset):
+    """Install dataset into pandas dataframes.
+       each file in a separate dataframe
+    """
+    #download(dataset)
+    installation_path = os.path.join(HOME_DIR,'{}.sqlite'.format(dataset))
+    install_sqlite(dataset, file=installation_path)
+    con = sqlite3.connect(installation_path)
+    cursor = con.cursor()
+    cursor.execute("SELECT name FROM sqlite_master WHERE type='table';")
+    tables = [t[0] for t in cursor.fetchall()]
```

Comment:
It's generally better to avoid single character variables names even in cases like this. How about `table` instead of `t`?